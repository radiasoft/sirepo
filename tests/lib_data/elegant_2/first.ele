&run_setup
  expand_for = "distr/distribution.sdds",
  lattice = "lattice.lte",
  use_beamline = "BL1",
&end

&run_control
&end

&sdds_beam
  input = "distr/distribution.sdds",
&end
