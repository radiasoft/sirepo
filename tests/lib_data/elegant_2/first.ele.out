
&global_settings
  mpi_io_write_buffer_size = 1048576,
&end

&run_setup
  semaphore_file = run_setup.semaphore,
  expand_for = "distr/distribution.sdds",
  lattice = "lattice.lte",
  use_beamline = "BL1",
&end

&run_control
&end

&sdds_beam
  input = "distr/distribution.sdds",
&end
